packages/vecosek-engine/vecosek-engine.0.0.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "Seb Mondet <seb@mondet.org>"
+authors: [
+  "Seb Mondet <seb@mondet.org>"
+]
+homepage: "https://gitlab.com/smondet/vecosek"
+bug-reports: "https://gitlab.com/smondet/vecosek/issues"
+dev-repo: "https://gitlab.com/smondet/vecosek.git"
+license: "ISC"
+version: "0.0.0"
+available: [ ocaml-version >= "4.03.0" ]
+build: [
+  ["ocaml" "please.ml" "configure"]
+  ["jbuilder" "build" "--only" "vecosek-engine" "--root" "." "-j" jobs "@install"]
```

Comment:
This line look quite different from the usual one. Is there any reason for that ? (see almost any other packages using jbuilder for examples)